webpack/ForemanTasks/Components/TaskDetails/TaskDetailsActions.js

```diff
@@ -38,29 +38,32 @@ export const taskReloadStart = (
   loading = false
 ) => {
   if (!timeoutId) {
-    timeoutId = setInterval(() => refetchTaskDetails(id, loading), 5000);
+    timeoutId = setInterval(
+      () => refetchTaskDetails(id, loading, timeoutId),
```

Comment:
what do you think about using the Interval middleware from core?
you can get the interval ID from the store, instead of passing it around from action to action

Reply:
Yeah I'll add it to redmine and do another PR

Reply:
https://github.com/theforeman/foreman-tasks/pull/564